src/main/java/app/bpartners/api/service/event/RefreshInvoiceSummaryTriggeredService.java

```diff
@@ -10,10 +14,22 @@
 @Service
 public class RefreshInvoiceSummaryTriggeredService
     implements Consumer<RefreshInvoiceSummaryTriggered> {
-  private final InvoiceSummaryService invoiceSummaryService;
+  private final UserService userService;
+  private final EventProducer eventProducer;
 
   @Override
   public void accept(RefreshInvoiceSummaryTriggered refreshInvoiceSummaryTriggered) {
-    invoiceSummaryService.updateInvoicesSummary();
+    getEnabledUsersIds();
```

Comment:
Rename this method. It actually says "get enabled users ids" but what it does is "sending InvoiceSummaryRefresh event" ...